builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,184 +112,339 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
+thirdweb offers a small number of chains from `@thirdweb/chains` and does not include Moonbeam networks in that list, so you'll need to specify the network details including chain ID and RPC URL. You can create a custom chain with [`defineChain`](https://portal.thirdweb.com/references/typescript/v5/defineChain){target=\_blank} as follows:
 
-```javascript
-import { ThirdwebProvider } from '@thirdweb-dev/react';
-import { Moonbeam } from '@thirdweb-dev/chains';
+=== "Moonbeam"
 
-const App = () => {
-  return (
-    <ThirdwebProvider activeChain={Moonbeam}>
-      <YourApp />
-    </ThirdwebProvider>
-  );
-};
+    ```typescript
+    import { defineChain } from 'thirdweb';
+    const moonbase = defineChain({
+    id: {{ networks.moonbeam.chain_id }},
+    rpc: '{{ networks.moonbeam.public_rpc_url }}',
+    })
+    ```
+
+=== "Moonriver"
+
+    ```typescript
+    import { defineChain } from 'thirdweb';
+    const moonbase = defineChain({
+    id: {{ networks.moonriver.chain_id }},
+    rpc: '{{ networks.moonriver.public_rpc_url }}',
+    })
+    ```
+
+=== "Moonbase Alpha"
+
+    ```typescript
+    import { defineChain } from 'thirdweb';
+    const moonbase = defineChain({
+    id: {{ networks.moonbase.chain_id }},
+    rpc: '{{ networks.moonbase.rpc_url }}',
+    })
+    ```
+
+## thirdweb sdk
+
+The following sections will provide an overview of fundamental methods of the thirdweb sdk and how to interact with them. Each code snippet will showcase the relevant import statements and demonstrate using the method in a typical scenario. This guide is intended to be a quick reference guide to the most common thirdweb methods that dApp developers will use. However, it does not include information on each and every thirdweb offering. For details on the entirety of thirdweb's offerings, be sure to visit the [thirdweb documentation site](https://portal.thirdweb.com/){target=\_blank}. For a comprehensive, step-by-step guide to building a dApp with thirdweb be sure to check out Moonbeam's [thirdweb tutorial in the tutorials section](/tutorials/eth-api/thirdweb). The following sections will cover everything from connecting wallets, to preparing transactions, and more.  
+
+### Accounts and Wallets
+
+thirdweb distinguishes between accounts and wallets in the sdk. In the eyes of the thirdweb SDK, an account always has a single blockchain address and can sign messages, transactions, and typed data, but it cannot be "connected" or "disconnected." In contrast, a wallet contains one or more accounts, can be connected or disconnected, and delegates the signing tasks to its accounts. 
+
+The below code snippet demonstrates how to initialize and connect a MetaMask wallet using the thirdweb SDK, then sign and send a transaction, retrieving the transaction hash. This process is applicable to any of the 300+ wallet connectors supported by the SDK.
+
+```typescript title="Initialize and Connect a Metamask Wallet"
```

Comment:
I think this should be extracted to a code snippet, so we can use it for the CN site

Reply:
I think titles should strictly be for file names. We should be explicitly telling users what the snippet does before the snippet,  so we don't need to do it here too